src/scss/abstracts/_button-classes.scss

```diff
@@ -0,0 +1,35 @@
+@import "../abstracts/button-mixins";
+
+.btn-close {
+  border: none;
+  width: 2.5rem;
+  height: 2.5rem;
+  background: url("/assets/images/icons/close.svg") center no-repeat;
```

Comment:
This assumes that this image file exists and can be served by the including application. Maybe we could use the [base64 encoded version of the image](https://www.hongkiat.com/blog/css-encoded-image/) here instead of relying on the external application?